docs/reference/commandline/service_create.md

```diff
@@ -47,6 +47,7 @@ Options:
       --name string                      Service name
       --network list                     Network attachments (default [])
       --no-healthcheck                   Disable any container-specified HEALTHCHECK
+      --placement-pref-add pref          Add a placement preference
```

Comment:
Hum, we tend to not have the `*-add` in the `create` subcommand (would be `--placement-pref` instead)